coverage_assessment.R

```diff
@@ -42,6 +42,140 @@ nut <- get_data()
 # Problem 3: Prevalence of acute undernutrition/malnutrition ----
 
 
+SEX <- ifelse(nut$sex == 1, "Male", "Female") ##Assigning male and female to sex
+
+OEDEMA <- ifelse(nut$oedema == 1, "Yes", "No") ## Assigning yes and No to oedema
+
+COUNTY <- nut$county ##Assigning nut$county to COUNTY
+
+Den <- sum(nut$age >=6 & nut$age<60, na.rm = TRUE) ## Assigning the total number of children between 6-60 to Den
+
+
+####NUMBER AND PREVALENCE OF MAM CASES
+
+
+Mod_MUAC <- with (nut, muac>= 11.5 & muac <12.5 & age >=6 & age<60) ## Assigning the moderate MUAC criteria for children between 6-60 for Mod_MUAC
+
+MAM <- table (SEX, OEDEMA, Mod_MUAC, COUNTY, useNA = "ifany") ## Assigning MAM to table 
+
+data.frame(MAM) ##CREATING A DATA FRAME FOR TOTAL MAM CASES
+
+
+MAM_num_F1 <- MAM ["Female", "No", "TRUE","Urban Montserrado"] ## TOTAL MAM FEMALES IN URBAN MONTSERRADO
```

Comment:
This is a unique and clever approach to this. But I am wondering whether you really know/appreciate what you were doing. Here, you are basically indexing a `table` object rather than a data.frame and in some ways I think you were lucky that the way you organised your table matches how the data.frame would have looked like.

In the future, it will be better from a learning and documentation perspective if you write this line of code this way:

```R
MAM_num_F1 <- MAM[SEX = "Female", OEDEMA = "No", Mod_MUAC = TRUE, COUNTY = "Urban Montserrado"]
```

I added the `table` dimension names to make it clear to the user of the code which variables I am indexing with. Notice also how in the `Mod_MUAC` variable I remove the quotes you used for `TRUE` or `FALSE`. Remember that `TRUE` or `FALSE` have special meaning in R and not all functions treat "TRUE" as equivalent to `TRUE`. Here you are lucky that this is the case for the `table` object but this is not always the case across other objects. It is better to learn the appropriate syntax that will be correct across all use cases.